jenkins/check-for-build.jenkinsfile

```diff
@@ -57,13 +57,15 @@ pipeline {
             agent {
                 docker {
                     label AGENT_X64
-                    image 'alpine:3'
+                    image 'docker/library/alpine:3'
+                    registryUrl 'https://public.ecr.aws/'
                     alwaysPull true
                 }
             }
             steps {
                 script {
                     dockerAgent = detectDockerAgent()
+                    sh("cat /etc/*release")
```

Comment:
Can we remove this cat? Is it required anywhere

Reply:
I was testing and yes this should be removed.